src/Render/Program.cs

```diff
@@ -52,17 +54,25 @@ internal static class Program {
 
 			var renderer = TemplateRenderer.CreateFromResource( "Templates.page.html" );
 			var formatted = await renderer.RenderAsync(
-				title: "TODO: Get Title",
+				title:GetTitle( doc ),
 				content: html
 			);
 
 			outputHtml.Write( formatted );
 		}
 
+		private static string GetTitle( MarkdownDocument doc ) {
+			var inline = doc.Descendants<HeadingBlock>().FirstOrDefault(
+				h => h.Level == 1
+			)?.Inline;
+			var titleLiteral = inline.Descendants<LiteralInline>().First();

```

Comment:
Hm, I wonder if this would work for

```
# Foo `bar` baz
```

?

For now maybe make this `Single` so that it crashes rather than making weird output.

```suggestion
			var titleLiteral = inline.Descendants<LiteralInline>().Single();
```

Reply:
I don't think making this work is urgent (e.g. urgent enough to do in this PR), I don't know how much work it'd be. Supporting basic titles for now is fine.

I opened an issue: #23

Reply:
So I just pushed a solution for this (it wasn't too hard). However the more interesting problem is code in the title.

The block ```<b>This is</b> `the title` [with link](www.google.ca)``` gives us the title text `This is with link`, because `Inline` objects all have a LiteralInline underneath, **except** `CodeInline`

Reply:
Above no longer applies. Going to leave this function in program.cs, will move somewhere better (or replace completely) later.
Going to work on plaintext renderer, that might replace this completely, will also help with some other stuff.